app/main.py

```diff
@@ -1,2 +1,74 @@
+from dataclasses import dataclass
+from typing import Hashable, Any
+
+
+@dataclass
+class Node:
+    key: Hashable
+    value: Any
+    key_hash: int
+
+
 class Dictionary:
-    pass
+    INITIAL_CAPACITY = 8
+    THRESHOLD = 2 / 3
```

Comment:
The `THRESHOLD` is calculated using integer division (`2 / 3`), which results in zero. This should be a float division (`2 / 3.0` or `2 / 3`) to correctly represent the threshold as a fraction.